test/tools/ossfuzz/abiV2FuzzerCommon.cpp

```diff
@@ -0,0 +1,13 @@
+#include <test/tools/ossfuzz/abiV2FuzzerCommon.h>
+
+using namespace dev::test::abiv2fuzzer;
+
+SolidityExecutionFramework::SolidityExecutionFramework(langutil::EVMVersion _evmVersion)
+{
+	m_evmVersion = _evmVersion;
+}
+
+SolidityExecutionFramework::SolidityExecutionFramework()
+{
+	m_evmVersion = langutil::EVMVersion();
```

Comment:
Do you plan to use a non-recent evm version?

Reply:
Not unless we also want to test against older versions, do we?

Reply:
We might...

Reply:
> We might...

So there is this constructor if we want to customize EVM version

https://github.com/ethereum/solidity/blob/a085e4721a6113cff8ede64c73b0b45e52fc1bec/test/tools/ossfuzz/abiV2FuzzerCommon.cpp#L5-L8